packages/ag-charts-community/src/chart/data/processors.ts

```diff
@@ -350,7 +350,7 @@ export function animationValidation(valueKeyIds?: string[]): ProcessorOutputProp
                 for (let i = 0; validation !== 0 && i < valuesDef.length; i++) {
                     const value = valuesDef[i];
 
-                    if (!valueKeyIds?.includes(value.id as string) === true) continue;
+                    if (!valueKeyIds?.includes(value.id as string)) continue;
```

Comment:
Minor - but `=== true` is better for engines because it's a bitwise operation vs. dynamic dispatch

Reply:
I think that in this very specific case it doesn't matter since we are converting the value to a boolean (`!`) meaning no additional casting will happen anyhow

Reply:
Ah yeah, it should be `valueKeyIds?.includes(value.id as string) !== true` 😅 But yes, semantically equivalent